--- misc/libxml2-2.15.3/xml2-config.in
+++ misc/build/libxml2-2.15.3/xml2-config.in
@@ -1,12 +1,17 @@
 #! /bin/sh
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-includedir=@includedir@
-libdir=@libdir@
+#prefix=@prefix@
+#exec_prefix=@exec_prefix@
+#includedir=@includedir@
+#libdir=@libdir@
 cflags=
 libs=
 
+prefix=${SOLARVERSION}/${INPATH}
+exec_prefix=${SOLARVERSION}/${INPATH}
+includedir=${SOLARVERSION}/${INPATH}/inc${UPDMINOREXT}/external
+libdir=${SOLARVERSION}/${INPATH}/lib${UPDMINOREXT}
+
 usage()
 {
     cat <<EOF
@@ -67,7 +72,8 @@
 	;;
 
     --cflags)
-        cflags="@XML_INCLUDEDIR@ @XML_CFLAGS@"
+        cflags="-I${includedir}"
+#        cflags="@XML_INCLUDEDIR@ @XML_CFLAGS@"
        	;;
 
     --libtool-libs)
@@ -82,16 +88,17 @@
        	;;
 
     --libs)
+        libs="-L${libdir} ${LIBXML2LIB} -lm"
         if [ "$2" = "--dynamic" ]; then
             shift
-            libs="@XML_LIBS@ @XML_PRIVATE_LIBS_NO_SHARED@"
-        else
-            libs="@XML_LIBS@ @XML_PRIVATE_LIBS@ @LIBS@"
-        fi
-
-        if [ "@libdir@" != "/usr/lib" -a "@libdir@" != "/usr/lib64" ]; then
-            libs="@XML_LIBDIR@ $libs"
+#            libs="@XML_LIBS@ @XML_PRIVATE_LIBS_NO_SHARED@"
+#        else
+#            libs="@XML_LIBS@ @XML_PRIVATE_LIBS@ @LIBS@"
         fi
+#
+#        if [ "@libdir@" != "/usr/lib" -a "@libdir@" != "/usr/lib64" ]; then
+#            libs="@XML_LIBDIR@ $libs"
+#        fi
         ;;
 
     *)
